### PR TITLE
Support Height at ActionSet, FactSet, Inputs(Number, Date, Time)

### PR DESCRIPTION
**Pull request by urmitchauhan**
## Description of changes made

- Changes and refactor at  FactSet, ActionSet and Inputs Number, Date, Time
- Remove cycle hierarchy layer
- Add Test case

## Sample Card

Old             	   |  Updated
:-------------------------:|:-------------------------:
Screenshot 2022-08-24 at 6 10 56 PM|Screenshot 2022-08-24 at 5 57 03 PM
Screenshot 2022-08-24 at 6 11 37 PM|Screenshot 2022-08-24 at 6 01 00 PM
Screenshot 2022-08-24 at 6 12 01 PM|Screenshot 2022-08-24 at 6 04 00 PM
Screenshot 2022-08-24 at 6 12 26 PM|Screenshot 2022-08-24 at 6 05 23 PM
Screenshot 2022-08-24 at 6 12 54 PM|Screenshot 2022-08-24 at 6 06 23 PM

## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.

At the very least please describe the issue you are addressing, and what your fix entails. 

## Sample Card
If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift [hunk 1]
```diff
@@ -22,10 +22,10 @@ class ActionSetRenderer: NSObject, BaseCardElementRendererProtocol {
         case .right: actionAlignment = .right
         default: actionAlignment = .left
         }
-        return renderView(actions: actions, aligned: actionAlignment, with: hostConfig, style: style, rootView: rootView, parentView: parentView, inputs: inputs, config: config)
+        return renderView(actionElement: nil, actions: actions, aligned: actionAlignment, with: hostConfig, style: style, rootView: rootView, parentView: parentView, inputs: inputs, config: config)
```
[the remark this answers is not in the document]
Comment (urmitchauhan) [reply source/macos/AdaptiveCards/AdaptiveCards/Renderers/ActionSetRenderer.swift:25]: Thanks ✅

File: source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift [hunk 2]
```diff
@@ -141,18 +155,25 @@ open class ACRNumericTextField: NSView, NSTextFieldDelegate {
...
+        contentStackView.addArrangedSubview(contentView)
+        contentView.addSubview(textField)
+        contentView.addSubview(stepper)
     }
     
     private func setupConstraints() {
-        textField.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
-        textField.topAnchor.constraint(equalTo: topAnchor).isActive = true
-        textField.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+        contentStackView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
```
Comment (rohshar6) [inline source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift:165]: can we add a helper functions for these **contentStackView.toFill(view: view)**, like we have in uc client for filling the whole view.

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRActionSetView.swift [hunk 3]
```diff
@@ -176,11 +189,20 @@ extension ACRActionSetView: ShowCardTargetHandlerDelegate {
             return
         }
         
+        func manageShowCardBottomLayoutConstraint(onCardVisible value: Bool) {
+            if value {
```
Comment (rohshar6) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRActionSetView.swift:193]:
```suggestion
           showCardStackViewBottomConstraint.constant = value ? 0 : exteriorPadding
```

Comment (urmitchauhan) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRActionSetView.swift:193]: Done ✅

File: source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift (hunk 2 above)
Comment (urmitchauhan) [reply source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift:165]: lets connect for this.

**Review by mukuagar — APPROVED** [review]

**Review by rohshar6 — APPROVED** [review]

**Pull request merged**